README.md

```diff
@@ -1,17 +1,61 @@
 # Northcoders News API
 
+## Project Aims ##
 An example API which accesses application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
 
-Your database will be PSQL, and you will interact with it using node-postgres.
+## Link To Hosted Project ##
+https://smarticles.onrender.com/api/articles/
 
-You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the relevent database name for that environment. Please ensure that these .env files are .gitignored.
+## Project Setup ##
+
+To replicate this project:
+
+Your database will be PSQL, this will be interacted with using node-postgres.
+
+#### Creating .env files ####
+You will need to create two .env files for your project: .env.test and .env.development. Into each, 
+add PGDATABASE=<>
+with the relevent database name for that environment. __Please ensure that these .env files are .gitignored.__ 
 
 This project requires several node dependicies listed in the package-lock.json/package.json files. After the env files are created run npm install.
 
-Creating the databases
-We'll have two databases in this project: one for real-looking dev data, and another for simpler test data.
+#### Creating the databases ####
+We'll have two databases in this project: one for development data, and another for simpler test data.
 Seeding data is provided:
-run setup.sql file to create the databases - psql -f ./db/setup.sql
-run run-seed.js - node ./db/seeds/run-seed.js
+run setup.sql file to create the databases 
+- -psql -f ./db/setup.sql
+run run-seed.js 
+- node ./db/seeds/run-seed.js
+
```

Comment:
Better to name the setup-dbs script, because that isn't dependent on the location of the terminal, and is built just to run the right files